From dfe88f832fe80f11077cb9d3de39e78f04a158a4 Mon Sep 17 00:00:00 2001
From: Linfeng Sun <linfeng.sun.dev@gmail.com>
Date: Mon, 10 Aug 2026 15:32:35 +0800
Subject: [PATCH 14897/15333] mailmap: update email address for Linfeng Sun

1. Update my university email to my personal Gmail, as the former will
   expire after graduation.
2. Fix a typo in my Gmail domain.

Link: https://lore.kernel.org/20260810073235.22980-1-linfeng.sun.dev@gmail.com
Signed-off-by: Linfeng Sun <linfeng.sun.dev@gmail.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---
 .mailmap | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/.mailmap b/.mailmap
index bf0a246e0ab9..d35e65d609d3 100644
--- a/.mailmap
+++ b/.mailmap
@@ -516,6 +516,8 @@ Leo Yan <leo.yan@linux.dev> <leo.yan@linaro.org>
 Liam R. Howlett <liam@infradead.org> <Liam.Howlett@oracle.com>
 Liam Mark <quic_lmark@quicinc.com> <lmark@codeaurora.org>
 Linas Vepstas <linas@austin.ibm.com>
+Linfeng Sun <linfeng.sun.dev@gmail.com> <slf@hdu.edu.cn>
+Linfeng Sun <linfeng.sun.dev@gmail.com> <linfeng.sun.dev@gamil.com>
 Linus Lüssing <linus.luessing@c0d3.blue> <linus.luessing@ascom.ch>
 Linus Lüssing <linus.luessing@c0d3.blue> <linus.luessing@web.de>
 Linus Lüssing <linus.luessing@c0d3.blue> <ll@simonwunderlich.de>
-- 
2.55.0

